setup.cfg

```diff
@@ -29,7 +31,6 @@ install_requires =
     packaging
     pandas
     pydantic
```

Comment:
This is the place to add version constraints, right? You are probably only compatible with Pydantic 2.x (that package changed a lot somewhat recently).

Reply:
Changed to `pydantic >= 2.0`